### PR TITLE
react router-dom setup

### LINKED ISSUE #22: React router setup
# Project setup
## FrontEnd

- install react-router using `npm i react-router-dom`.
- From `react-router-dom` import:
  - BrowserRouter as Router,
  - Switch
  - Route
  - Link
- Wrapp the Router around app component. 
- create an empty files for all pages

### PR DESCRIPTION
**Pull request by yasmeenattallah**
Fix #22

- [x] install react router dom  ``npm install react-router-dom``
- [x] import ``{ BrowserRouter,Switch,Route}``
- [x] wrapp `` <Router> </Router> `` around app component 
- [x] insert a new file path : ``` client/src/utils/router.constant.js ``` to add router path in constant way 
- [x] import router constant file in app.js and add the pathes in ``` <Route> ```
- [x] make an empty file for pages 
- [x] create a dummy skeleton for each page and  render them on the app  page

### CODE REVIEW

**Review by imansedky5 — APPROVED** [review]

File: client/src/utils/router.constant.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const HOME_PAGE = '/';
+export const BLOGS = '/blog';
+export const SIGNIN_PAGE = '/signin';
+export const SIGNUP_PAGE = '/signup';
+export const PROFILE = '/profile';
+export const CONTACTS = '/contact';
+export const BOOK = '/book';
+export const SERVICES = '/services';
+export const DASHBOARD = '/dashboard';
+export const DASHBOARD_APPOINTMENT = 'dashboard/appointment	';
```
Comment (mFlaifel) [inline client/src/utils/router.constant.js:10]: Good job,
- it's the first time see this pattern, can you explain  how the component will render (please  create a dummy skeleton on each page and show us how to render the page),
- don't forget the `404` page.

**Review by AdhamHaisami — DISMISSED** [review]

File: client/src/utils/router.constant.js (hunk 1 above)
Comment (yasmeenattallah) [reply client/src/utils/router.constant.js:10]: Thanks, 

- the component will render in ``` <Route> ```  component when the Route path ``` Ex :  <Route path= {Home_page} > ```
match the current URL, it renders its children ( ``` <Home/> ``` component )

- ok I'll create Now

Comment (imansedky5) [inline client/src/utils/router.constant.js:10]: remove the spaces from the end of this constant so it will be '`dashboard/appointment'`

**Review by imansedky5 — CHANGES_REQUESTED** [review]

**Review by khamis-hesham — DISMISSED** [review]

**Review by mFlaifel — APPROVED** [review]
well done

**Review by khamis-hesham — APPROVED** [review]

**Review by AdhamHaisami — APPROVED** [review]

**Review by imansedky5 — APPROVED** [review]

**Pull request merged**